postgres/assets/configuration/spec.yaml

```
@@ -98,19 +98,32 @@ files:
         display_default: null
     - name: relations
       description: |
-        The list of relations/tables must be specified here to track per-relation (table) metrics.
+        The list of relations/tables must be specified here to track per-relation (table, index , view, etc.) metrics.
         If enabled, `dbname` should be specified to collect database-specific relations metrics.
         You can either specify a single relation by its exact name in 'relation_name' or use a regex to track metrics
         from all matching relations (useful in cases where relation names are dynamically generated, e.g. TimescaleDB).
         Each relation generates many metrics (10 + 10 per index).
+
         By default all schemas are included. To track relations from specific schemas only,
         you can specify the `schemas` attribute and provide a list of schemas to use for filtering.
 
+        Size metrics are collected only for ordinary tables, index metrics are collected for user indexes the rest of
+        metrics are collected for user tables except lock metrics that are collected for all relation types
+        (table, index , view, etc.).
```

```suggestion
        Size metrics are collected only for ordinary tables. Index metrics are collected only for user indexes. Lock metrics are are collected for all relation types (table, index , view, etc.). The rest of the metrics are collected only for user tables.
```
In attempting to fix the run-on sentence, I might have altered the meaning (I added a couple "only"s). Please check. At very least, fix the run-ons, and change "user tables except lock metrics that" to "user tables, except lock metrics, which"